sos/report/plugins/sunbeam.py

```diff
@@ -57,7 +55,9 @@ def setup(self):
             manifests = yaml.safe_load(manifest_raw['output'])
             for manifest in manifests:
                 self.add_cmd_output(
-                    f'sunbeam manifest show --id {manifest["manifestid"]}')
+                    f'sunbeam manifest show {manifest["manifestid"]}',
```

Comment:
I don't think any sunbeam command should be run outside the context of the `sunbeam_user`. If it works, it's not by design.

Reply:
Thankf for that, I'll get that updated on the next update on this PR

Reply:
This should be updated in the updated PR now